dts/bindings/dma/gd,gd32-dma-v1.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2022, TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  GD32 DMA controller with FIFO
+
+  channel: Select channel for data transmitting
+
+  slot: Select peripheral to connect DMA
+
+  config: A 32bit mask specifying the DMA channel configuration
+    - bit 6-7:   Direction  (see dma.h)
+                 - 0x0: MEMORY to MEMORY
+                 - 0x1: MEMORY to PERIPH
+                 - 0x2: PERIPH to MEMORY
+                 - 0x3: reserved for PERIPH to PERIPH
+
+    - bit 9:     Peripheral address increase
+                 - 0x0: no address increment between transfers
+                 - 0x1: increment address between transfers
+
+    - bit 10:    Memory address increase
+                 - 0x0: no address increase between transfers
+                 - 0x1: increase address between transfers
+
+    - bit 11-12: Peripheral data width
+                 - 0x0: 8 bits
+                 - 0x1: 16 bits
+                 - 0x2: 32 bits
+                 - 0x3: reserved
+
+    - bit 13-14: Memory data width
+                 - 0x0: 8 bits
+                 - 0x1: 16 bits
+                 - 0x2: 32 bits
+                 - 0x3: reserved
+
+    - bit 15:    Peripheral Increment Offset Size
+                 - 0x0: offset size is linked to the peripheral bus width
+                 - 0x1: offset size is fixed to 4 (32-bit alignment)
+
+    - bit 16-17: Priority
+                 - 0x0: low
+                 - 0x1: medium
+                 - 0x2: high
+                 - 0x3: very high
+
+  fifo-threshold: A 32bit bitfield value specifying FIFO threshold
+    - bit 0-1:   DMA FIFO threshold selection
```

Comment:
What is the purpose of this cell? Is this a trigger? What FIFO is this? Is it part of peripheral side, DMA side? At devicetre tree bindings you must state the meaning in a way that maintainers don't need to look at datasheet. The obvious cells like channel and slot don't require to much attention but platform specific we need good explanation.

Reply:
Thank you for checking.
I update the description about the `fifo-threshold`.
This cell is a configuration for the burst transfer feature (but not yet supported), 
clarified how it works with other parameters.